api/src/main/java/org/openmrs/Person.java

```diff
@@ -325,6 +327,21 @@ public void setCauseOfDeath(Concept causeOfDeath) {
 		this.causeOfDeath = causeOfDeath;
 	}
 	
+	/**
+	 * @return cause of person's death non coded
+	 * @since Trunk-5405
+	 */
+	public String getCauseOfDeathNonCoded() {
+		return this.causeOfDeathNonCoded;
+	}
+	
+	/**
```

Comment:
This one also needs the since annotation

Reply:
I think you commented on the same code segment as above. 

Does the setter and getter need a since annotation. Is that the intention of the comment

Reply:
I set the version to 2.9.0 since 2.8.0 is already relaesed and the discussion on Talks (https://talk.openmrs.org/t/mark-patient-as-dead/6279/29) suggested that it should be 2.8.0 or 2.9.0. Can you please verify @dkayiwa since I am not completely sure what the right value should be?

Reply:
You can read the next version from here: https://github.com/openmrs/openmrs-core/blob/master/pom.xml#L17

Reply:
So 2.2.0 is the next version?

Reply:
yes